README.md

```diff
@@ -58,6 +58,7 @@ The following settings are supported:
   
 Since 0.3.0:
 * `xml.format.spaceBeforeEmptyCloseTag`: Set to `true` to insert space before the end of a self closing tag.
+* `xml.problems.noGrammar`: Set the type of warning for grammar constraint issues. Default is `hint`.
```

Comment:
Set the message severity when a document has no associated grammar. Default is `hint`.